src/org/parosproxy/paros/network/HttpHeader.java

```diff
@@ -104,13 +105,13 @@
     protected static final String p_VERSION = "(HTTP/\\d+\\.\\d+)";
     protected static final String p_STATUS_CODE = "(\\d{3})";
     protected static final String p_REASON_PHRASE = "(" + p_TEXT + ")";
-    protected String mStartLine = "";
-    protected String mMsgHeader = "";
-    protected boolean mMalformedHeader = false;
-    protected Hashtable<String, Vector<String>> mHeaderFields = new Hashtable<>();
-    protected int mContentLength = -1;
-    protected String mLineDelimiter = CRLF;
-    protected String mVersion = HttpHeader.HTTP10;
+    protected String mStartLine;
+    protected String mMsgHeader;
+    protected boolean mMalformedHeader;
+    protected Hashtable<String, Vector<String>> mHeaderFields;
+    protected int mContentLength;
+    protected String mLineDelimiter;

```

Comment:
If CRLF is declared then this variable is later set to that const, why not just use the const throughout?